bin/install-jetpack.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+pushd jetpack
+
+pnpm_version=$(npx -c 'echo "$npm_package_engines_pnpm"')
+
+# Remove pnpm hoisting config if on CI
+if [[ -n "${CI:-}" ]]; then
+  sed -i.bk '/hoist/d' .npmrc
```

Comment:
This is the extreme option: reset the hoisting configuration back to the pnpm defaults. We might be able to go lighter and keep the Jetpack configuration but add what ever we need to build, that is we might only need to add a few `hoist-pattern[]=*@something/something*` calls.

Reply:
Also, I tried overriding the npm configuration via the command line but from what I found the project level configs in `.npmrc` are overridable from global or user npm configs. The only option is to modify the .npmrc file.